Time: 28/03/2021 11:50:36 PM
-----------------------------------------------------------
Message: An error occurred while updating the entries. See the inner exception for details.
StackTrace:    at System.Data.Entity.Internal.InternalContext.SaveChanges()
   at System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   at System.Data.Entity.DbContext.SaveChanges()
   at DataEditor.myClass.crmManagment.add(crm Crm, String path) in C:\Users\mosha\OneDrive\Desktop\mvc\DataEditor\DataEditor\myClass\crmManagment.cs:line 58
Source: EntityFramework
TargetSite: Int32 SaveChanges()
-----------------------------------------------------------

Time: 28/03/2021 11:55:21 PM
-----------------------------------------------------------
Message: An error occurred while updating the entries. See the inner exception for details.
StackTrace:    at System.Data.Entity.Internal.InternalContext.SaveChanges()
   at System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   at System.Data.Entity.DbContext.SaveChanges()
   at DataEditor.myClass.crmManagment.add(crm Crm, String path) in C:\Users\mosha\OneDrive\Desktop\mvc\DataEditor\DataEditor\myClass\crmManagment.cs:line 58
Source: EntityFramework
TargetSite: Int32 SaveChanges()
-----------------------------------------------------------

Time: 28/03/2021 11:55:35 PM
-----------------------------------------------------------
Message: An error occurred while updating the entries. See the inner exception for details.
StackTrace:    at System.Data.Entity.Internal.InternalContext.SaveChanges()
   at System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   at System.Data.Entity.DbContext.SaveChanges()
   at DataEditor.myClass.crmManagment.add(crm Crm, String path) in C:\Users\mosha\OneDrive\Desktop\mvc\DataEditor\DataEditor\myClass\crmManagment.cs:line 58
Source: EntityFramework
TargetSite: Int32 SaveChanges()
-----------------------------------------------------------

Time: 28/03/2021 11:57:31 PM
-----------------------------------------------------------
Message: An error occurred while updating the entries. See the inner exception for details.
StackTrace:    at System.Data.Entity.Internal.InternalContext.SaveChanges()
   at System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   at System.Data.Entity.DbContext.SaveChanges()
   at DataEditor.myClass.crmManagment.add(crm Crm, String path) in C:\Users\mosha\OneDrive\Desktop\mvc\DataEditor\DataEditor\myClass\crmManagment.cs:line 58
Source: EntityFramework
TargetSite: Int32 SaveChanges()
-----------------------------------------------------------

